site/data/theme-colors.yml

```
@@ -1,3 +1,9 @@
+- name: primary
+  hex: "#ab0520"
+  contrast_color: black
+- name: secondary
+  hex: "#0c234b"
+  contrast_color: black;
```

I think these need a white contrast color? See https://review.digital.arizona.edu/arizona-bootstrap/issue/1613/docs/5.0/utilities/background/.

```suggestion
- name: primary
  hex: "#ab0520"
  contrast_color: white
- name: secondary
  hex: "#0c234b"
  contrast_color: white
```